CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ If you are looking for the Ledger Releasing and Versioning Process then you can
 
 ## 8.7
 
+- Remove the library libs/cardano-ledger-pretty
```

Comment:
This time it is the right CHANGELOG :slightly_smiling_face: , but not the right version.

Next version that will be releaed that is affected is 8.8

```suggestion
## 8.8

- Remove the library libs/cardano-ledger-pretty

## 8.7
```